### PR TITLE
Add support for multi embeds

### PR DESCRIPTION
**Pull request by VelvetThePanda**
Discord recently added the ability to send multiple embeds in normal messages. This (assuming git hasn't failed me again) will add support for that.

### CODE REVIEW

**Comment by VelvetThePanda** [comment]
If someone could look through and make sure I haven't screwed everything up what'd be nice; I'll be testing in the meanwhile

**Comment by VelvetThePanda** [comment]
ok it would help if I actually read documentation

**Comment by IDoEverything** [comment]
You shouldn't remove the `WithEmbed` method, that would be a very breaking change

**Comment by VelvetThePanda** [comment]
I need to redo this anyway. Discord is throwing 400s. Docs say array, API says dictionary

**Comment by VelvetThePanda** [comment]
GitHub wtf

**Comment by VelvetThePanda** [comment]
The magic of an IDE that couldn't care less about github thinks

**Comment by IDoEverything** [comment]
Instead of removing the `Embed` property entirely, you should make it set the collection to just the embed. Breaking changes should be avoided as much as possible. Also this should be added to edit message

**Comment by IDoEverything** [comment]
Though the `get` on the `Embed` property could be an issue

**Comment by VelvetThePanda** [comment]
> Also this should be added to edit message

What do you mean

**Comment by IDoEverything** [comment]
You can also do multiple embeds on edit message now

**Comment by VelvetThePanda** [comment]
Yes and that should be supported as well

File: DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs [hunk 1]
```diff
@@ -168,16 +183,35 @@ public DiscordMessageBuilder HasTTS(bool isTTS)
...
             this.Embed = embed;
             return this;
         }
 
+        /// <summary>
+        /// Appends an embed to the current builder.
+        /// </summary>
+        /// <param name="embed">The embed that should be appended.</param>
+        /// <returns>The current builder to be chained.</returns>
+        public DiscordMessageBuilder AddEmbed(DiscordEmbed embed) => this.WithEmbed(embed);
```
Comment (IDoEverything) [inline DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs:201]: If you're changing `WithEmbed` this shouldn't point to that

Comment (VelvetThePanda) [reply DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs:201]: I'll fix that in a second

**Review by IDoEverything — APPROVED** [review]
Seems fine at a glance. If something still breaks I claim no responsibility whatsoever

**Comment by VelvetThePanda** [comment]
ok I missed a spot and so did you @IDoEverything

**Comment by Neuheit** [comment]
Can you link the documentation for this in the API docs? I can't seem to find it.

**Comment by IDoEverything** [comment]
https://github.com/discord/discord-api-docs/commit/fd6501898c0827a04e7ab2eb49190f3fa9871028 they really should update the website more often

**Pull request merged**